hooks/post/30-upgrade.rb

```diff
@@ -38,6 +38,11 @@ def import_subscriptions
   Kafo::Helpers.execute('foreman-rake katello:upgrades:2.4:import_subscriptions')
 end
 
+def remove_elasticsearch
+  Kafo::Helpers.execute('rpm -e elasticsearch lucene3 ruby193-rubygem-tire cigar') if system('rpm -q elasticsearch')
```

Comment:
right, it should work in that scenario which is the right scenario to test in :)